lib/common/client/middleware/response/snakecase.rb

```diff
@@ -0,0 +1,33 @@
+module Common
+  module Client
+    module Middleware
+      module Response
+        class Snakecase < Faraday::Response::Middleware
+
+          def on_complete(env)
+            if env.response_headers['content-type'] =~ /\bjson/
```

Comment:
👍 i did experiment with doing that and had issues, but that might've been for other errors.